python-bindings/pythonbindings.cpp

```diff
@@ -362,6 +374,7 @@ PYBIND11_MODULE(blspy, m)
         .def(
             "__bytes__",
             [](const G1Element &ele) {
+	        PythonGIL release_lock;
                 vector<uint8_t> out = ele.Serialize();
                 py::bytes ans = py::bytes(
```

Comment:
you have to acquire the lock again before constructing `py::bytes`.

Reply:
You are correct. Fixed.